docs/api.md

```diff
@@ -0,0 +1,10 @@
+# API
+
+## Builds
+
+### POST `/api/builds/:app`
+
+<a name="api-build-api"></a>
+
+Builds the docker image of an application from the master branch with a unique
+tag and upload it to the configured Docker registry.
```

Comment:
```suggestion
Builds the docker image of an application from the `master` branch with a unique
tag and uploads it to the configured Docker registry.
```
Also, I believe it is not especially `master` but rather "the first step defined in the workflow".